lib/controllers/messages.ts

```diff
@@ -308,10 +298,15 @@ export function CreateMessageHandler(
       // tslint:disable-next-line:no-object-mutation
       context.bindings.createdMessage = createdMessageEvent;
 
+      const responsePayload: CreatedMessageResponse = {
+        dry_run: false
+      };
+
       // redirect the client to the message resource
       return ResponseSuccessRedirectToResource(
         retrievedMessage,
-        `/api/v1/messages/${fiscalCode}/${message.id}`
+        `/api/v1/messages/${fiscalCode}/${message.id}`,
```

Comment:
better to not hardcode API paths (expecially true for version)

Reply:
they have to be hardcoded _somewhere_ :)

Reply:
also that code is not part of this PR, I'll add a reminder in the backlog

Reply:
https://www.pivotaltracker.com/story/show/151217639